docs/actions.md

```diff
@@ -193,6 +193,35 @@ Key | Description | Possible Values
 *url* | Url of Nexus IQ server |
 *organization-id* | Organization UID like "222de33e8005408a844c12eab952c9b0" |
 
+## OpsGenie
+
+??? note "Set up OpsGenie and get a token"
+
+    1. Go to your Opsgenie and select Teams from menu.
+    2. Select your team to access your team dashboard.
+    3. Select Integrations from left navigation.
+    4. Select Add Integration.
+    5. Select API Integration.
+    6. Copy `API Key`.
+    7. When done with all configurations, select Save Integration to enable the integration.
+
+    See more details here: [Set up an integrated tool for Opsgenie](https://support.atlassian.com/opsgenie/docs/set-up-an-integrated-tool/).
+
+!!! note
+      `Postee` requires an API key from an API integration. An API key from the API Key Management tab will produce this 403 error 
```

Comment:
I'm not sure if I understand this, do you mean the key the user can get from OpsGenie isn't enough in some way?

Reply:
I see - so you mean the highlighted API key from the "API key management" block will not work but instead the user needs to get this key from "team's integration"? 

I think it's very valuable to include this information as you mentioned. I'm just trying to better word it. Could you share that link for the forum where you found this? I'll try to better word it.

Reply:
I just saw your previous comments and got the link https://community.atlassian.com/t5/Opsgenie-questions/OpsGenie-API-Create-alert-Authentication-problem/qaq-p/1531047 - I believe this is the one you were referring to?

Reply:
Made a suggestion - I think I understand now. Thanks.